samples/csharp_dotnetcore/57.teams-conversation-bot/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "MicrosoftAppType": "",
-  "MicrosoftAppId": "",
-  "MicrosoftAppPassword": "",
+  "MicrosoftAppId": "96bb8032-091a-4220-b610-6b8450cd6be2",
+  "MicrosoftAppPassword": "NSQ8Q~aPLEgv5~T2SXv0-FrywVhp41L.oI~vHcyg",
```

Comment:
Be careful with the credentials.

Reply:
I see you reverted this, however, the commit that exposes the credentials should not be part of the PR. We should clean the history.